docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
+When added to your bundler's plugin list, the plugin deploys your bundled applicatioon to a remote Zephyr Cloud instance.
+
+## Deployment lifecycle
+The deployment lifecycle can be split into three stages:
+
+*   [Initial bundling](#initial-bundling)
+*   [Fact gathering](#fact-gathering)
+*   [Snapshot upload and tagging](#snapshot-upload-and-tagging)
+
+### Initial bundling
+The application is bundled according to your bundler's configuration.
+The Zephyr Cloud bundler plugin runs as the last step in the bundling process, initiating the next stage of the process.
+
+### Fact gathering
+The bundler plugin first attempts to gather facts about the application.
```

Comment:
Not sure we want to call this "Fact gathering" while it is technically accurate I'm not sure if thats what we want to call it.  @valorkin may have other suggestions, but perhaps something related to "Environment Assessment" or "Workspace Assessment" or "Project Assessment"

Reply:
I believe it's "Environment Assessment"

Reply:
Changed the terminology used in [4b97ebc](https://github.com/ZephyrCloudIO/zephyr-documentation/pull/5/commits/4b97ebc4b5a7faa4c146688380a4e6dc4b217796)